getting-started/creating-hypertables.md

```diff
@@ -71,4 +59,3 @@ examples of using TimescaleDB's standard SQL interface, please see our
 [setup]: /getting-started/setup
 [API Reference]: /api#create_hypertable
 [use pages]: /using-timescaledb
-[best practices]: /using-timescaledb/hypertables#best-practices
```

Comment:
Are you planning to remove the section on space partitions from `hypertables.md`?

Reply:
Nah, I wasn't thinking of doing that since I guess we do have to talk about it. Do you think it should be removed?